prisma/schema.prisma

```diff
@@ -350,8 +350,9 @@ model Metadata {
 
 model User {
   id    String @id @default(cuid())
-  email String @unique
-  name  String
+  email String?
+  name  String @unique
+  displayName String?
```

Comment:
Maybe a personal preference or a bit nitpicky, but I feel like "name" and "displayName" have the same meaning. I think switching `name` to `username` will suffice